### DISCUSSION TITLE
Connect-PnPOnline reponds with Could not load file or assembly

### DISCUSSION THREAD

**Discussion by cakeowels** [pnp/powershell#457] [Q&A]
I am using Powershell v7.1.3 and I was working perfectly fine with PnP.Powershell v1.4 loaded, I was able to modernize SPOnline sites etc yesterday. I came to my Powershell console today and Connect-PnPOnline reponds with Could not load file or assembly  'Microsoft.Extensions.DependencyInjection, Version=2.2.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'. The system cannot find the file specified.

I tried Register-PnPManagementShellAccess and that just freezes with no response, and I have to crash out of Powershell.

Any thoughts what might be happening here?

**Comment by bGlasson** [comment]
Same issue.... though I had working _-UseWebLogin_ code that broke with _The type initializer for 'Microsoft.SharePoint.Client.ClientContextExtensions' threw an exception_ ... I tried switching to _-Interactive_  and got same error as OP.
